web/blog/2024-09-03-OS-builders-interview-with-tim-jones-pgboss.md

```diff
@@ -0,0 +1,102 @@
+---
+title: 'The Faces Behind Open Source Projects: Tim Jones and pg-boss'
+authors: [milica]
+image: /img/os-interviews/pg-boss-banner.jpeg
+tags: [webdev, auth, react, django, tutorial, full-stack]
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ImgWithCaption from './components/ImgWithCaption'
+
+![wasps interviewing Tim](/img/os-interviews/pg-boss-banner.jpeg)
+
+We’re launching a new series of posts where we'll sit down with the folks who help us run our projects without expecting anyting in return. Yes, we're talking about open-source maintainers and builders, the people who dedicate their free time to make tech better. This is our way to say "Thank you!" to all of those who help us build and improve [Wasp](https://wasp-lang.dev/), as well as shape the webdev ecosystem.
+
```

Comment:
maybe worth explicitly mentioning that Wasp Jobs are powered by pg-boss under the hood?